narwhals/_pandas_like/series.py

```diff
@@ -1031,6 +1031,53 @@ def rank(
 
         return self._from_native_series(ranked_series)
 
+    def hist(
+        self: Self,
+        bins: list[float | int] | None = None,
+        *,
+        bin_count: int | None = None,
+        include_category: bool = True,
+        include_breakpoint: bool = True,
```

Comment:
Remove default values in compliant implementations
```suggestion
        bins: list[float | int] | None,
        *,
        bin_count: int | None,
        include_category: bool,
        include_breakpoint: bool,
```